create_aio_app/template/{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/__main__.py

```diff
@@ -2,8 +2,10 @@
 import uvloop
 {%- endif %}
 from aiohttp import web
-
 from .app import init_app
+    {%- if cookiecutter.use_sentry == 'y' %}
```

Comment:
Don't add trailing spaces.
```suggestion
{%- if cookiecutter.use_sentry == 'y' %}
```

Reply:
trailing is deleted